spec/services/doc_auth/lexis_nexis/requests/true_id_request_spec.rb

```diff
@@ -3,19 +3,24 @@
 RSpec.describe DocAuth::LexisNexis::Requests::TrueIdRequest do
   let(:image_source) { nil }
   let(:account_id) { 'test_account' }
-  let(:workflow) { nil }
   let(:base_url) { 'https://lexis.nexis.example.com' }
+  let(:workflow) { subject.send(:workflow) }
```

Comment:
I'm curious why you made this change?  Not against it, just curious about it.

Reply:
@eileen-nava , to verify workflows being used (which is determined by a private method)